build/index.js

```diff
@@ -155,7 +155,7 @@ function postLocalFileLinks($, doc) {
     const href = element.attribs.href;
 
     // This test is merely here to quickly bail if there's no hope to find the
-    // image as a local file link. `Image.findByURL()` is fast but there are
+    // image as a local file link. `Image.findByURLWithFallback()` is fast but there are
     // a LOT of hyperlinks throughout the content and this simple if statement
     // means we can skip 99% of the links, so it's presumed to be worth it.
```

Comment:
Small thing but, is this entire comment still valid with the new function used here?

Reply:
I don't have a proper benchmark to attest for the change of performance. There are, at worst, two lookups and a few instructions more but the order of magnitude should not change. I'll remove the specific mention of the function since I'm not as confident as the previous author. Thanks for spotting :)